newIDE/app/src/Utils/GetObjectByName.js

```diff
@@ -15,3 +15,18 @@ export default function getObjectByName(
 
   return null;
 }
+
+export const hasObjectWithName = (
+  globalObjectsContainer: gdObjectsContainer | null,
+  objectsContainer?: ?gdObjectsContainer,
+  objectName: string
+): boolean => {
```

Comment:
`ObjectsContainersList` should be used instead. You can pass a `ProjectScopedContainersAccessor`.